### PR TITLE
Add depth estimation UI elements 

### PR DESCRIPTION
**Pull request by sayakpaul**
Related issue: https://github.com/huggingface/hub-docs/issues/473

Cc: @NielsRogge @nateraw @merveenoyan

### CODE REVIEW

**Comment by osanseviero** [comment]
Would it be possible to split the PR in two?
1. The UI elements (e.g. the tag, tasks files)
2. The task page.

The first @gary149 @mishig25 and I can review.
The second @merveenoyan can review :)

**Comment by sayakpaul** [comment]
> Would it be possible to split the PR in two?
> 
> 1. The UI elements (e.g. the tag, tasks taxonomy, etc)
> 2. The task page.
> 
> The first @gary149 @mishig25 and I can review. The second @merveenoyan can review :)

@osanseviero done.

File: tasks/src/video-classification/about.md
Comment (osanseviero) [inline tasks/src/video-classification/about.md:1]: This seems unrelated to this PR

File: tasks/src/video-classification/data.ts
Comment (osanseviero) [inline tasks/src/video-classification/data.ts:1]: This seems unrelated to this PR

**Review by osanseviero — CHANGES_REQUESTED** [review]
There are some changes unrelated to this PR, such as the assets under tasks for depth estimation and the video classification task page

**Comment by sayakpaul** [comment]
> There are some changes unrelated to this PR, such as the assets under tasks for depth estimation and the video classification task page

Took care of it. Sorry.

**Review by osanseviero — APPROVED** [review]
Looks good to me! cc @mishig25 @gary149 to approve the icon element (I have not build moon landing with this)

(I think you can remove tasks/assets/depth-estimation/depth-estimation-output.png and the other image from this PR as well)

**Comment by sayakpaul** [comment]
> I think you can remove tasks/assets/depth-estimation/depth-estimation-output.png and the other image from this PR as well)

@osanseviero removed.

**Comment by mishig25** [comment]
@gary this is how the icon looks:

File: js/src/lib/components/Icons/IconDepthEstimation.svelte
Comment (mishig25) [inline js/src/lib/components/Icons/IconDepthEstimation.svelte:1]: two points:
1. Could you add the script tag & class={classNames} as in other icons. Example here https://github.com/huggingface/hub-docs/blob/12c9de46be6a3d9bbb54658040144eb028ebd593/js/src/lib/components/Icons/IconCaretDown.svelte#L1-L6
2. Run `npm run format` which will format this file

Otherwise, looks great to me !

Comment (sayakpaul) [reply js/src/lib/components/Icons/IconDepthEstimation.svelte]: Thanks, @mishig25! Just pushed the changes. LMKWYT.

**Comment by mishig25** [comment]
lgtm! let's megre it once @gary149 approves the icon choice 👍

File: js/src/lib/components/Icons/IconDepthEstimation.svelte
Comment (gary149) [inline js/src/lib/components/Icons/IconDepthEstimation.svelte:8]:
```suggestion
		fill="currentColor"
```

**Comment by mishig25** [comment]
@gary149 good to merge ?

**Comment by gary149** [comment]
> @gary149 good to merge ?

yes

**Comment by coyotte508** [comment]
I think the `depth-estimation` folder is missing.

See https://github.com/huggingface/moon-landing/pull/4527 - the PR to update the docs is failing

**Comment by coyotte508** [comment]
Oh I see there is https://github.com/huggingface/hub-docs/pull/513

But in general I think it's good if the main branch is always in an "ok" state - for example this PR could have added an empty file with the minimum exports needed to not break the build.

**Comment by sayakpaul** [comment]
My apologies. Sorry.

**Pull request merged**
